posts/2024-01-24_wyg-zooma/README.md

```diff
@@ -0,0 +1,45 @@
+
+The goal was to form an in-real-life DAO whose members shared common goals and interests—using blockchain technology with a token to pay for goods and services and vote on governance matters. While many people participated and were enthusiastic about creating a shared society, zôÖma admits the experiment was early: no one was interested in interacting with the tech. “90% of people, rural or not, found it too complicated,” he admits. “We were a bit naive, but it was fascinating nonetheless.”
+
+## A Fascination for Human Behavior
+
+zôÖma has been an ardent student of human behavior since childhood. His parents taught him early on the value of philanthropy and working with people in need. He’s set up several joint liability companies, non-profits, and NGOs to experiment with finding new and better ways to organize society, and one of the things he loves most about web3 is its “experimental” nature. He’s encouraged by how far the industry has come since he received his first bitcoins in partial payment for a website in 2014. “That turned out to be a really expensive website for my customer,” he laughs. He never expected such broad adoption of Bitcoin and a technology that “inspired a whole generation of engineers to experiment with new things.”
+
+Like most creative types, zôÖma is used to spinning many plates in the air, overseeing La Suite du Monde while working as a freelance designer, front-end dev, and Artistic Director for an independent French record label. “Before entering the world of engineers, I founded and managed a collective for 12 years, which brought together artists from all disciplines, hackers, designers, tinkerers, to build some interesting projects.” La Suite de Monde allowed him to explore his passion for finding new approaches to social coordination first-hand. “I explored very radical things,” he says, “like the notion of “accepted by default” where anyone could use the collective budget by expressing their desire to do so three times. I wouldn’t recommend this,” he laughs, “but the experiments were fascinating and still serve me today in my work.”
+
+One really interesting trait about zôÖma is how he harnesses the creative and analytical sides of himself with equal application. Most people are predominantly right-brained or left-brained, yet, zôÖma is ‘ambidextrous’ in this regard. He’s a designer who’s created large-scale artistic events, cultural tours of Paris, and an award-winning independent movie documenting French artist and Bitcoin advocate Pascal Boyart, [The Underground Sistine Chapel](http://www.the-chapel.art), (which you should definitely check out!). Yet he’s also passionate about engineering and the future of cooperatives. He’s detail-driven and ambitious, taking his team at Teritori from two to 18 (14 full-time teammates and four part-time).
```

Comment:
```suggestion
One really interesting trait about zôÖma is how he harnesses the creative and analytical sides of himself with equal application. Most people are predominantly right-brained or left-brained, yet, zôÖma is ‘ambidextrous’ in this regard. He’s a designer who’s created large-scale artistic events, cultural tours of Paris, and an award-winning independent movie documenting French artist and Bitcoin advocate Pascal Boyart, [The Underground Sistine Chapel](http://www.the-chapel.art), (which you should definitely check out!). Yet he’s also passionate about engineering and the future of cooperatives. He’s detail-driven and ambitious, taking his team at Teritori from 2 to 18 (14 full-time teammates and four part-time).
```

Reply:
@waymobetta We follow the [AP guidelines](https://guides.18f.gov/content-guide/our-style/numbers-and-percentages/#:~:text=Generally%20speaking%2C%20we%20follow%20the,10th%20or%20greater%20with%20numerals) when it comes to writing out numbers. In body copy, we write out numbers one to nine in full.